src/inspector/calculatedAttributes/types.ts

```diff
@@ -0,0 +1,6 @@
+import { Result } from './errors';
+import { Standards } from './standards';
+
+export type ParserResult<T> = Partial<Record<Standards, Result<T>>>;
```

Comment:
I think we should perhaps remove this partial. Only reason the standard is not in the record, is because it would throw error. But that would make the whole record invalid anyways.